### ISSUE TITLE
NCDA CHW Questionnaire - Antenatal Care [4h]

### ISSUE THREAD

**Issue by adamhstewart** [TIP-Global-Health/eheza-app#745]
## Issue

<img width="228" alt="Cursor_and_NCDA_–_Figma" src="https://github.com/TIP-Global-Health/eheza-app/assets/688507/f947a8a8-292f-450c-b6ce-0cb3bf63a1f4">

The antenatal care screen documents antenatal care for the mother of the child during her pregnancy (i.e. we need a connected mother).

## Tasks
- Implement the fronted rendering of the page with questions as shown in design.
- Implement the following logic: 
    - The first statement and question should only be shown if the mother does not have 4 ANC visits in E-Heza
    - If the answer to the above conditional question is no, provide the question "How many ANC visits did the mother receive?"
    - If the answer to the above conditional question is no, provide the statement "Provide counseling to the mother on the importance of regular ANC visits." (not pictured).
    - If the answer to "Did the mother receive  Iron, folic acid, and MMS?" is "no" OR (the answer is yes AND the answer to "Has she taken it as per guidance" is "no", provide the statement "Provide the counseling to the mother on the consequences that may occur to the mother and the baby and referral the mother to the HC to receive the Iron/Folic Acid/MMS."
    -  Only show this activity if the questions have not been asked before in a previous activity.
  

Design page:
https://www.figma.com/file/l0bvoD2VyyFc146VVIYWOQ/NCDA?type=design&node-id=2-3&t=dDpvst1vV7hU7yzj-0

**Comment by anvmn** [comment]
@adamhstewart 
> - If the answer to the above conditional question is no, provide the question "How many ANC visits did the mother receive?"
> - If the answer to the above conditional question is no, provide the statement "Provide counseling to the mother on the importance of regular ANC visits." (not pictured).

First question answer is a number, not boolean.
So, what should be condition for question 2?

**Comment by adamhstewart** [comment]
@anvmn ^^ less than 4

**Comment by Diama1** [comment]
@anvmn @adamhstewart I have made an excel sheet that has all the counseling part for all the questions in different categories. https://docs.google.com/spreadsheets/d/1RSoW7M7mVvt0QehrKTXG8MnWe82yzUBXjOzkgyAQB4g/edit?usp=sharing

**Comment by Diama1** [comment]
@anvmn  @adamhstewart 
|  Question       |     Counseling    |
|  ---------------     |   -------------------- |
|  How many ANC standard visits did the mother recieve? | Provide the counseling on the consequences that may occur to her and the baby if she doesn't attend ANC visit as per guidance. |
| Has she taken it as per guidance(CHW observes)? |  provide the counseling to the mother on the consequences that may occur to the mother and the baby and referral the mother to the HC to receive the Iron/Folic Acid/MMS .|

**Comment by anvmn** [comment]
@adamhstewart 

![Selection_2033](https://github.com/TIP-Global-Health/eheza-app/assets/17882245/6db7deed-f0f4-4644-80f1-bf164f3345b8)

As we discussed, this implies that mother was tracking this specific pregnancy on E-heza.
With current mode of children imports when adding villages, the chances for this are close to 0%.
The effort however to track that pregnancy from data is substantial.
When working on previous developments for NCDA, we decided not to implement this.
Must this be mandatory this time?
